crates/resvg/src/main.rs

```diff
@@ -526,13 +525,13 @@ fn parse_args() -> Result<Args, String> {
     let mut default_size = usvg::Size::from_wh(100.0, 100.0).unwrap();
     if let (Some(w), Some(h)) = (args.width, args.height) {
         default_size = usvg::Size::from_wh(w as f32, h as f32).unwrap();
-        fit_to = FitTo::Size(w, h);
+        fit_to = FitTo::Size(w as f32, h as f32);
```

Comment:
Would it make sense for these arguments to themselves be floats? I think *probably* not, because they need to be positive and finite. But creating a massive image can still cause issues.